ci/browser-tests/cypress/integration/cost_items.spec.js

```diff
@@ -0,0 +1,210 @@
+describe("Cost items, totals and materials views", function() {
+  let costItemOid
```

Comment:
should use context, not let variables that are set... as the cypress code runs it just creates a model of what to do in the test.

```cy.wrap(someExpression).as("myThing")```

that can later be used with `this.myThing`